### DISCUSSION TITLE
RSBot v2.9

### DISCUSSION THREAD

**Discussion by myildirimofficial** [myildirimofficial/RSBot#644] [Announcements]
## What's Changed
* Fix config saving on auto mastery up for [Run while bot stoped] option by @SDClowen in https://github.com/SDClowen/RSBot/pull/609
* Fix detecting when got any item on item storage box window by @SDClowen in https://github.com/SDClowen/RSBot/pull/610
* Fix null reference exception when player join to active party by @SDClowen in https://github.com/SDClowen/RSBot/pull/611
* Added some pickup options for pickup manager by @SDClowen in https://github.com/SDClowen/RSBot/pull/612
* Fix cave issue #556 by @SDClowen in https://github.com/SDClowen/RSBot/pull/613
* Fix entity positions while drawing on map plugin by @SDClowen in https://github.com/SDClowen/RSBot/pull/614
* Fixed a bug that caused the bot to sometimes run twice and load multi… by @SDClowen in https://github.com/SDClowen/RSBot/pull/615
* Sometimes, the client start button was disabling. by @SDClowen in https://github.com/SDClowen/RSBot/pull/616
* Optimized defaultbot loop by @SDClowen in https://github.com/SDClowen/RSBot/pull/617
* Fix auth and serverlist by @SDClowen in https://github.com/SDClowen/RSBot/pull/618
* Fix: Collision disabled by user error. by @SDClowen in https://github.com/SDClowen/RSBot/pull/619
* Fix parsing on npctalk, also its fixing pandora bug by @SDClowen in https://github.com/SDClowen/RSBot/pull/620
* Fix parsing error when selecting portals and deselecting npc while tu… by @SDClowen in https://github.com/SDClowen/RSBot/pull/621
* Fix inventory updating after alchemy success by @SDClowen in https://github.com/SDClowen/RSBot/pull/622
* Fix: The kill counter counting in twos on statistics plugin by @SDClowen in https://github.com/SDClowen/RSBot/pull/623
* Fix: Freezing while char teleport on crowded players area on rigid by @SDClowen in https://github.com/SDClowen/RSBot/pull/624
* Fix skill buffs saving and using on protection plugin #525 by @SDClowen in https://github.com/SDClowen/RSBot/pull/625
* Fix: There was no have any SpecialtyBag inventory control when pickup… by @SDClowen in https://github.com/SDClowen/RSBot/pull/626
* Fix: Plugin/Command-Handler was not working because of packet parsing by @SDClowen in https://github.com/SDClowen/RSBot/pull/627
* Fix: auto join any party via contains title by @SDClowen in https://github.com/SDClowen/RSBot/pull/628
* Fixed issue #563 by @SDClowen in https://github.com/SDClowen/RSBot/pull/629
* Fix some parsing errors & exceptions by @SDClowen in https://github.com/SDClowen/RSBot/pull/642
* [Feature] Add trader bot base by @ngoedde in https://github.com/SDClowen/RSBot/pull/635
![image](https://user-images.githubusercontent.com/14323838/236635962-a9a4aa69-ddf2-4fb7-b12e-20b44694cce7.png)

![image](https://user-images.githubusercontent.com/14323838/236635967-8ace8f59-6fee-45e2-a19f-901f7d606428.png)

**Full Changelog**: https://github.com/SDClowen/RSBot/compare/v2.8...v2.9

<hr /><em>This discussion was created from the release <a href='https://github.com/SDClowen/RSBot/releases/tag/v2.9'>RSBot v2.9</a>.</em>

**Comment by junepham6** [comment]
When you start using it, you will disconnect

**Comment by ynan3000** [comment]
There are some suggestions:
1- Add storing items in guild storage.
2- Allow the bot to open skills.
3- Send notification after finding a specific item.
4- Log the drop distribution in party to know who got the item.
5- Add counting players in the area and classify them to job and non job players,
6- Show the number of thiefs and traders/ hunters in the area.
7- Make the players allowed to add plugins to the bot.
8- Allow the bot to update the client.
9- Create RSBot Manager.

**Comment by KillingKarma** [comment]
RSbot 2.9.3 version has been upgraded and seems to work well. But there are still many errors and limitations. In the settings for picking up items and filtering items, rsbot does not work. The Lure section in RSbot also doesn't work. I'm playing Silkroad Origin VTC (Vietnam)

**Comment by KillingKarma** [comment]
Your Discord page is broken. It disappeared from my Discord app. Looks like someone reported your page

**Comment by goco58** [comment]
sayfa kaderine  terkedilmis discorda da  burada da maalesef
